src/docfx/restore/Restore.cs

```diff
@@ -43,7 +43,8 @@ public static bool RestoreDocset(string docsetPath, string? outputPath, CommandL
             {
                 // load configuration from current entry or fallback repository
                 var configLoader = new ConfigLoader(errorLog);
-                var (errors, config, buildOptions, packageResolver, fileResolver) = configLoader.Load(disposables, docsetPath, outputPath, options, fetchOptions);
+                var (errors, config, buildOptions, packageResolver, fileResolver) = configLoader.Load(
```

Comment:
How about putting `configLoader.Load(` into the next line?

Reply:
The change is follow the rule "If it is long parameter, try put parameters to a new line, if it is still too long, break each parameter into its own line", should I update the rule?

Reply:
I guess the rule is specific for method definition?
Take this [change](https://github.com/dotnet/docfx/pull/6134/files#diff-0b7306773d16a50c3dae5a64c992125aL146-L147) for instance, assignment might be a different story

Reply:
The scope is not specified for the rule, add @yufeih to discuss the change for the case.